namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java

```diff
@@ -674,4 +676,10 @@ private RemotingCommand getClientConfigs(ChannelHandlerContext ctx, RemotingComm
         return response;
     }
 
+    private RemotingCommand healthCheck(ChannelHandlerContext ctx, RemotingCommand request) {
```

Comment:
Can we just build a command line script, querying existing broker status info? The server just respond with memory info to the cli,